### ISSUE TITLE
No option to generate new deploy key

### ISSUE THREAD

**Issue by oerp-odoo** [docker/hub-feedback#2386]
So old bot user access got lost that was connected with docker hub. It was using deploy key (public key) provided here:

![Selection_1245](https://github.com/docker/hub-feedback/assets/7812986/c69aac43-1088-470d-818b-e4e618c08d0d)

The problem now is that I can't use that deploy key anymore, because it is already used up on user I can't access. Github says such key is already used.

But I dont see a way to generate new deploy key. To me this looks like insecure behavior by docker hub. What if key in use was compromised, why can't I drop that key and get new one?..

I tried deleting repository, and creating new. Yet when new repo is created I still see same key..:)

**Comment by gdibernardo** [comment]
Hi @oerp-odoo,

Have you sorted this out? Is this still an issue for you?

**Comment by oerp-odoo** [comment]
Yes, I just switched to manually pushing to docker hub. And consider migrating to other container providers, because hub feels unmaintained with all kinds of strange quirks
